client/state/selectors/get-past-billing-transaction-error.js

```diff
@@ -0,0 +1,21 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import { get } from 'lodash';
+
+/**
+ * Returns a past billing transaction.
+ * Returns null if the billing transactions have not been fetched yet, or there is no transaction with that ID.
+ *
+ * @param  {Object}  state   Global state tree
+ * @param  {String}  id      ID of the transaction
+ * @return {?Object}         The transaction object
```

Comment:
I find this a little confusing, because looking at the tests we only return `bool` values. Looking at the code, it seems we expect error objects. Probably we need to update tests to reflect this.